modules/android/pages/vector-search.adoc

```diff
@@ -58,12 +58,65 @@ This representation encodes how a ML model understands the input or inputs provi
 When a model considers the features of a given input as similar, the distance between the vector embeddings will be short.
 Vector embeddings are stored within embedded vector indexes.
 
+The current supported formats for Vector embeddings are:
+
+* An array of 32-bit floats.
+
+* A Base64 string that encodes the binary representation of an array of 32-bit float numbers in https://en.wikipedia.org/wiki/Endianness[Little-endian] format.
+
 === About Vector Indexes
 
 Vector Indexes are used to store and manage vector representations of content in the form of vector embeddings.
 You can use Vector Indexes to efficiently retrieve vectors, similar to a target vector.
 Before use, a Vector Index needs to be trained to compute the centroids and parameters for encoding the vectors. 
 Training is configured to commence automatically on the first Vector Search query when there are enough vectors to be trained based on the `minimum-training-size` configuration.
+If training is triggered during a query, the query may take longer to return results.
+If a query is executed against the index before it is trained, a full scan of the vectors will be performed. 
+
+NOTE: If there are insufficient vectors in the database for training, a warning message will be logged, indicating the required number of vectors.
+
+=== About Lazy Vector Indexes
+
+You can use Lazy Vector Indexes(lazy index) to use a ML model that is not available locally on the device and to create vector indexes without having vector embeddings in the documents.
+Lazy indexing is an asynchronous process that provides developers full control during the following:
+
+* When to update the index.
+
+* The number of vectors to update to the index.
+
+* Cancelling or skipping select indexes when the model is unavailable or has failed.
+
+NOTE: Updating in lazy index is an independent process from saving document operations.
+
+You can use lazy indexing for situations when the ML model may be unreachable or not functioning, scheduling the index updating process on demand and handling situations during failed update attempts by skipping the process and retrying the index update process at a later point when the ML model is available.
+
+IMPORTANT: Lazy index is not an automatic process, you will need to manually schedule the index updates. 
```

Comment:
Suggest you be more explicit about what proces you mean here. Do you mean Updating a lazy index?  Also consider combining this info with the previous note

Reply:
Actioned